notebooks/en/pipeline_notus_instructions_preferences_legal.ipynb

```diff
@@ -0,0 +1,804 @@
+{
```

Comment:
<p>switch "we need to mingle a little bit with the dataset." to something like "we need to rename some of the columns in our dataset"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>